tests/functional/test_install_vcs_git.py

```diff
@@ -10,6 +10,26 @@
 from tests.lib.local_repos import local_checkout
 
 
+def _checkout_into_temp(url_info, temp_dir, egg=None):
```

Comment:
Make `scheme` a keyword argument, defaulting to https, to further reduce repetition...

Reply:
Rename the function to show that github is a special snowflake?